### PR TITLE
DDF-3215 updates to plugins docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates documentation for catalog plugins

#### Who is reviewing it? 

@mcalcote @vinamartin @emmberk @ahoffer @Lambeaux @jrnorth @brendan-hofmann 

#### Choose 2 committers to review/merge the PR. 
@clockard
@coyotesqrl
@lessarderic
@pklinef
@ricklarsen - Documentation
@shaundmorris
@stustison
#### How should this be tested? (List steps with links to updated documentation)

review new content.

#### What are the relevant tickets?

[DDF-3215](https://codice.atlassian.net/browse/DDF-3215)

#### Screenshots (if appropriate)

[jdocumentation.pdf](https://github.com/codice/ddf/files/1232476/jdocumentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/jdocs/content/_plugins/content-uri-access-plugin-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+:type: plugin
+:status: published
+:title: Content URI Access Plugin
+:link: _content_uri_access_plugin
+:plugintypes: access
+:summary: Prevents a Metacard's resource URI from being overridden by an incoming UpdateRequest.
+
+The Content URI Access Plugin prevents a Metacard's resource URI from being overridden by an incoming UpdateRequest.
+
```
Comment (ahoffer) [inline distribution/docs/src/main/jdocs/content/_plugins/content-uri-access-plugin-contents.adoc]: Is it worth saying something about why we have the content uri plugin? We don't want someone to update an a metacard they have access to with a resource URI that points to a resource that is classified above their level.

File: distribution/docs/src/main/resources/_contents/_tables/Claims_Handler_Manager-table-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+|LDAP Bind User DN
+|ldapBindUserDn
+|String
+|DN of the user to bind with LDAP. This user should have the ability to verify passwords and read attributes for any user.
+|cn=admin
+
+|LDAP Bind User Password
+|password
+|Password
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/_tables/Claims_Handler_Manager-table-contents.adoc]: This isn't a `String`?

File: distribution/docs/src/main/jdocs/content/_tables/Claims_Handler_Manager-table-contents.adoc [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+|LDAP Bind User DN
+|ldapBindUserDn
+|String
+|DN of the user to bind with LDAP. This user should have the ability to verify passwords and read attributes for any user.
+|cn=admin
+
+|LDAP Bind User Password
+|password
+|Password
```
Comment (mcalcote) [inline distribution/docs/src/main/jdocs/content/_tables/Claims_Handler_Manager-table-contents.adoc]: Is this actually a `Password` type?

File: distribution/docs/src/main/resources/_contents/_tables/Claims_Handler_Manager-table-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/Claims_Handler_Manager-table-contents.adoc]: Source: https://github.com/codice/ddf/blob/master/platform/security/sts/security-sts-ldaplogin/src/main/resources/OSGI-INF/metatype/metatype.xml#L54

File: distribution/docs/src/main/jdocs/content/_tables/Claims_Handler_Manager-table-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_tables/Claims_Handler_Manager-table-contents.adoc]: Source: https://github.com/codice/ddf/blob/master/platform/security/sts/security-sts-ldaplogin/src/main/resources/OSGI-INF/metatype/metatype.xml#L54

File: distribution/docs/src/main/jdocs/content/_plugins/content-uri-access-plugin-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_plugins/content-uri-access-plugin-contents.adoc]: I'm not against it

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**